### DISCUSSION TITLE
**Community feedback welcome!** Hubs on DigitalOcean

### DISCUSSION THREAD

**Discussion by actionTurtle** [mozilla/hubs#3843] [Hubs Cloud Questions] [answered]
I have been trying to get hubs running on digital ocean... The walk through says its in alpha still... 

There has been some really strange behavior... I have been building a droplet from the Hubs marketplace image on digital ocean.. I had to destroy and re-create the droplets a few times because email was not working... 

But then I started getting a connection refused error... Now even if its a new droplet... and run through all the command prompts... I get no connection when trying to go to the site...

Anyone else have this problem? Anyone else trying to get hubs working on digital ocean? I hate amazon... lol

**Comment by robinkwilson** [comment, marked as answer]
If you have issues with emails, you can run the setup script multiple times (rather than destroying the droplet). I also ran into email issues when I was deploying to DO.

Unfortunately, the team doesn't have capacity right now to support Digital Ocean deploys. However, we've had a lot of people in the community successfully go through this process and might be able to help you. I'll leave this open for community members to pipe in.

Have you joined our discord community server? You can see if any of the community members can help with your deployment. I'd also add some screenshots of what you're seeing so people can help you better :)
